test: error handling updates

MinervaEn opened the pull request:
- removed individual listeners and added a before each listener instead
- added a utils testlog file to standardize the logging to console.
- added verbose to console logs.

mattmazzola commented on cypress/support/utils/testlog.js:
```diff
@@ -0,0 +1,34 @@
+}
+
+function printDoubleLine(length) {
+    console.log(`${now} ${"*".repeat(length)}`);
```
Looks like a typo: `${now}` -> `${now()}`

mattmazzola commented on cypress/support/utils/testlog.js:
```diff
@@ -0,0 +1,34 @@
+}
+
+function testHeader(testName) {
+    console.log(`${now()} ${"*".repeat(60)}`);
```
Re-use the value of `now()` rather than recomputing

mattmazzola commented on cypress/support/utils/testlog.js:
```diff
@@ -0,0 +1,34 @@
+    return Cypress.moment().format("MMMDD-HH:mm.ss");
+}
+
+export { now, printDoubleLine, printResult, testHeader, reportError, printStep }
```
Keep consistent naming.  This file is should be a logger, and re-use same javascript vocabulary as `log` instead of `print`.  In the web printing refers to printing on paper or to a device so I would prefer to keep the prefix `log`.

Also, instead of `reportError` I would prefer to use the builtin `console.error` or `console.warn`  I think they already print stack traces

MinervaEn commented:
All green! 
-removed individual listeners and added a before each listener instead
-added a utils testlog file to standardize the logging to console.
-added verbose to console logs.
-renamed function names to use  the prefix log instead of print.

mattmazzola commented:
:tada: This PR is included in version 0.293.5 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@conversationlearner/ui)
- [GitHub release](https://github.com/Microsoft/ConversationLearner-UI/releases/tag/v0.293.5)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
